docs/challenge-000.md

```diff
@@ -41,11 +39,19 @@ Next clone the repo locally and set up a dev container for your exi
 4. The VS Code window will reload and start building the dev container. A progress notification provides status updates. You only have to build a dev container the first time you open it; opening the folder after the first successful build will be much quicker.
 5. After the build completes, VS Code will automatically connect to the container.
 You can now interact with your project in VS Code just as you could when opening the project locally. From now on, when you open the project folder, VS Code will automatically pick up and reuse your dev container configuration.
+6. Build and run the starter code, setting a breakpoint to see how the 
 
 ## References
 - [VS Code Developing inside a Container](https://code.visualstudio.com/docs/remote/containers)
 
+## Challenge Completion Criteria
+To successfully complete this challenge you should have:
 
+- [ ] Installed the necessary prerequisites 
+- [ ] A dev container running in VS Code
+- [ ] The starter code running in the dev container
 
-## Challenge Completion Criteria
-By the end of this challenge you should have the necessary prerequisites installed and your own version of the repo in a dev container running in VS Code.
\ No newline at end of file
+## Next Steps
+Now that you have your dev environment setup and the starter code running in a dev container in VS Code, it's time to begin processing data from Cosmos.
+
+Proceed to [Challenge 1](https://github.com/cse-southeast-devcrew/openhack-durable-entities/blob/shbanerj/37/docs/challenge-001.md).
```

Comment:
need relative path link here else you'll be updating this again later ;)

Reply:
fixed!